src/table/table.component.ts

```diff
@@ -393,7 +395,7 @@ import { I18n } from "./../i18n/i18n.module";
 			</ng-template>
 			<tr *ngIf="this.model.isLoading">
 				<td class="table_loading-indicator">
-					<ibm-static-icon icon="loading_rows" size="lg"></ibm-static-icon>
+
```

Comment:
No replacement?

Reply:
We can drop in the small loader for now ... carbon doesn't really have a specific "table loading" state, so we need to make it more generic, and (ideally) get some design guidance